src/config.json

```diff
@@ -39,73 +39,53 @@
     "rules": [
         {
             "name": "Asking For Help",
-            "triggers": ["1", "ask", "help"],
-            "description": "Actually ask your question, don't just ask for \"help\"."
-        },
-        {
-            "name": "Explain What's Wrong",
-            "triggers": ["2", "work", "explain"],
-            "description": "Don't ask why your code doesn't \"work\"."
+            "triggers": ["0", "ask", "help", "details"],
+            "description": "Help us to help you, instead of just saying \"my code doesn't work\" or \"can someone help me.\""
```

Comment:
The full stop (or period, Americans smh...) should be between the two quotes.

Reply:
it is

Reply:
you're looking at the red